introduction.tex

```diff
@@ -1,2 +1,40 @@
+SymPy is an full featured computer algebra system (CAS) written in the Python
+programming language. It is open source, licensed under the extremely
+permissive BSD license, which allows anyone to reuse SymPy or its source code,
+even for commercial purposes. SymPy was started by Ond\v{r}ej \v{C}ert\'{\i}k
+in 2005, and it has since grown into a large open source project, with over
+500 contributors.
+% citation?
 
+SymPy is written entirely in the Python programming language,
```

Comment:
I agree. We don't mention SymEngine anywhere in the paper currently. It might be worth mentioning in the "conclusion and future work" section. SymEngine ought to have its own paper (if Ondrej wants to do that).